platform-dependent-components/problem-solver/cxx/verification-module/test/testStructures/notEnoughSpecifiedObjects/templates.scs

```diff
@@ -0,0 +1,18 @@
+test_template = [*
+_specified_object _=> nrel_annotation:: _t;;
+_t _<- annotation; sc_text;;
+*];;
+
+test_template -> rrel_key_sc_element: _specified_object;;
+
+test_specification
+<- sc_node_structure;
+=> nrel_template: test_template;
+=> nrel_decomposition:
+  {
+  test_class_a;
+  test_class_b
```

Comment:
probably another approach should be implemented, because [pr](https://github.com/ostis-ai/ostis-standard/pull/432) does not have checked classes in specification decomposition